elements/lisk-p2p/src/p2p.ts

```diff
@@ -268,19 +242,11 @@ export class P2P extends EventEmitter {
 		this._config = config;
 		this._isActive = false;
 		this._hasConnected = false;
+		this._peerServer = undefined;
```

Comment:
we can use `?` for  `private _peerServer?: PeerServer;` then we don't have to initialize it to `undefined`